project/Dependencies.scala

```diff
@@ -119,21 +120,28 @@ object Dependencies {
 
   val springDeps = Seq(
     jacksonDataFormatProto,
-    "org.springframework.boot" % "spring-boot" % SpringVersion,
-    "org.springframework.boot" % "spring-boot-starter-webflux" % SpringVersion)
+    "org.springframework" % "spring-web" % SpringFrameworkVersion,
+    "org.springframework" % "spring-webflux" % SpringFrameworkVersion,
+    "org.springframework" % "spring-webmvc" % SpringFrameworkVersion,
+    "org.springframework.boot" % "spring-boot" % SpringBootVersion,
+    "org.springframework.boot" % "spring-boot-starter" % SpringBootVersion,
+    "org.springframework.boot" % "spring-boot-starter-json" % SpringBootVersion,
+    "jakarta.websocket" % "jakarta.websocket-api" % "2.1.0")
```

Comment:
just curious, where is this one needed?

Reply:
Previously, we were including `spring-boot-starter-webflux` that was bringing in a few dependencies including this one. 

Without this jar, we get `ClassNotFoundException: jakarta.websocket.Endpoint`. 

But I still need to drill it. I'm not getting this error again on a sample that I'm testing. So not yet clear why (yet).

Reply:
Something very odd. 

I added 2.1.0 as a dependency here, but in a project that I was using to test, I have it explicitly set to 2.0.0. 

It seems that 2.0.0 works, but 2.1.0 not. When a remove the deps to 2.0.0 and let it use the one it gets transitively, it starts to fail.